packages/conversation/src/models/v1/conversation-delete-event/conversation-delete-event.ts

```diff
@@ -0,0 +1,33 @@
+import { Conversation } from '../conversation';
+
+/**
+ * This callback is sent when a conversation between the subscribed app and a contact is deleted.
+ */
+export interface ConversationDeleteEvent {
+
+  /** Id of the subscribed app. */
+  app_id?: string;
+  /** Timestamp marking when the channel callback was accepted/received by the Conversation API. */
+  accepted_time?: string;
+  /** Timestamp of the event as provided by the underlying channels. */
+  event_time?: string;
+  /** The project ID of the app which has subscribed for the callback. */
+  project_id?: string;
+  /** Context-dependent metadata. Refer to specific callback\'s documentation for exact information provided. */
+  message_metadata?: string;
+  /** The value provided in field correlation_id of a send message request. */
+  correlation_id?: string;
+  /** @see ConversationNotification */
+  conversation_delete_notification?: ConversationNotification;
+
+  trigger: 'CONVERSATION_DELETE';
```

Comment:
Maybe makes sense to make enum for types of triggers to have a unified list in one place

Reply:
It's the type `WebhookTrigger` that defines all the possible values

Reply:
can you do smth like that: `trigger: WebhookTrigger.CONVERSATION_DELETE` then?

Reply:
No, this is not an enum but a type. I've created an new interface to ensure the trigger value matches the type